### PR TITLE
switched away from grunt

### PR DESCRIPTION
**Pull request by mzedel**
FYI @olehermanse @aleksandrychev - just in case you want to get rid of some of the dependencies
this _should_ work - however I only used it without a token...

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thank you for submitting a PR! Maybe @olehermanse can review this?

File: README.md [hunk 1]
```diff
@@ -55,8 +53,8 @@ docker run -p 80:80 -p 81:81 --volume /path/to/volume:/home/proxy --name cfbs-we
...
-* [build-index](https://github.com/cfengine/build-index) - Index of modules
-* [build-website](https://github.com/cfengine/build-website) - Website
-* [cfbs](https://github.com/cfengine/cfbs) - Command line client
-* [modules](https://github.com/cfengine/modules) - Official modules provided by the CFEngine team
-* [module-template](https://github.com/cfengine/build-example) - Template for creating new modules
+- [build-index](https://github.com/cfengine/build-index) - Index of modules
+- [build-website](https://github.com/cfengine/build-website) - Website
+- [cfbs](https://github.com/cfengine/cfbs) - Command line client
+- [modules](https://github.com/cfengine/modules) - Official modules provided by the CFEngine team
+- [module-template](https://github.com/cfengine/build-example) - Template for creating new modules
```
Comment (olehermanse) [inline README.md:60]: Please don't change this, it's copy-pasted in multiple repos, so easier to keep in sync if you don't.

File: build.js [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+import less from 'less';
+import { extname } from 'path';
+import { minify } from 'uglify-js';
+
+import { CONTENT_PATH_PREFIX, getAllFiles, PAGES_INDEX_PATH, LESS_LOCATIONS, parseFrontMatter, writeFileSyncInFolder } from './common.js';
+
+if (!process.env.hasOwnProperty('GITHUB_USERNAME_TOKEN')) {
+  console.error('GITHUB_USERNAME_TOKEN env variable is not set. Format is: username@token');
+  process.exit(1);
+}
```
Comment (olehermanse) [inline build.js:12]: This change breaks just running `docker build` without a token, which is quite useful for me (and also the reason why GH action is failing).

**Review by olehermanse — CHANGES_REQUESTED** [review]
Thanks for working on this, looks like we're almost there :)

File: README.md (hunk 1 above)
Comment (mzedel) [reply README.md]: should now again be in sync with the rest 👌

File: build.js (hunk 2 above)
Comment (mzedel) [reply build.js]: Ahhh, I first thought you welcomed the change to be reminded to set the token locally... Now you will just get a log message, as I assume the token will be set in CI... 👌

**Review by aleksandrychev — APPROVED** [review]
looks good, thanks!

**Review by olehermanse — APPROVED** [review]
I tested it locally and seems okay :)

**Pull request merged**